gloo/docs/introduction/faq.md

```diff
@@ -6,7 +6,7 @@ description: Frequently Asked Questions
 
 ## Gloo API Gateway
 
-This section covers some of the highl-level questions we commonly see in community meetings, on our [public slack](https://slack.solo.io), or on [GitHub issues](https://github.com/solo-io/gloo/issues). Gloo is an API Gateway built on top of [Envoy Proxy](https://envoyproxy.io) that comes with a simple yet powerful control plane for managing Envoy as an edge ingress, API Gateway, or service proxy. Gloo's control plane is built on a plugin model that enables extension and customization depending on your environment and comes with an out of the box Discovery plugin that can discover services running VMs, registered in Consul, running in Kubernetes, or deployed on a public cloud including Functions running in a Cloud Functions environment.  The Envoy community moves fast and no two operational environments are identical, so we built Gloo with this flexibility in mind.
```

Comment:
the diff here isn't very helpful, but the only change here was the typo at the beginning:
> This section covers some of the **highl**-level questions